std/debug/index.zig

```diff
@@ -283,7 +283,8 @@ fn printSourceAtAddressWindows(di: *DebugInfo, out_stream: var, relocated_addres
     var coff_section: *coff.Section = undefined;
     const mod_index = for (di.sect_contribs) |sect_contrib| {
         if (sect_contrib.Section >= di.coff.sections.len) continue;
```

Comment:
If it's 1-based then this comparison is incorrect - it would have to be `sect_contrib.Section > di.coff.sections.len`.

Reply:
Oops, obviously. It's not used anywhere else.

Reply:
Thinking about it, I wonder if it's even possible to have a `Section` index that's outside the array ? It would probably mean the pdb is invalid.